src/Reducer/cartReducer.js

```diff
@@ -0,0 +1,27 @@
+const cartReducer = (state, action) => {
+  switch (action.type) {
+    case "ADD_TO_CART":
+      return { ...state, cart: [...state.cart, { ...action.payload, qty: 1 }] };
+
+    case "REMOVE_FROM_CART":
+      return {
+        ...state,
+        cart: state.cart.filter((item) => item._id !== action.payload._id),
+      };
+
+    case "CHANGE_CART_QUANTITY":
+      return {
+        ...state,
+        cart: state.cart.filter((item) =>
+          item._id === action.payload._id
+            ? (item.qty = action.payload.qty)
+            : item.qty
+        ),
+      };
```

Comment:
You don't need to manually handle cart operations. Using MockBee, you just have to make API calls for cart operations, and you will get the updated cart as the response.